Modules/CraftQueue/UI.lua

```diff
@@ -1603,7 +1603,8 @@ function CraftSim.CRAFTQ.UI:UpdateQueueDisplay()
         local firstRow = queueTab.content.craftList.activeRows[1]
         local craftButton = firstRow.columns[11].craftButton --[[@as GGUI.Button]]
         local button = craftButton.frame --[[@as Button]]
-        queueTab.content.craftNextButton:SetEnabled(button:IsEnabled())
+        local isCrafting = C_TradeSkillUI.IsRecipeRepeating()

```

Comment:
Just tested this api call: C_TradeSkillUI.IsRecipeRepeating()
It seems to be only true if the recipe is crafted with a count of more than 2
If its just crafted once it returns false

Maybe there is another api call to be used here?

Reply:
I couldn't find another API call for this. Indeed, when I click "Next: Craft," the button only disables after the second craft.  
Maybe registering an event on button click?